data.php

```diff
@@ -0,0 +1,34 @@
+<?php
+session_start();
+require 'functions.php';
+$db = connectToDatabase();
+
+$name_input= $_POST["name"];
+$address_input= $_POST["address"];
+$rating_input= $_POST["rating"];
+$local_brewery_input= $_POST["local_brewery"];
+$picture_input= $_POST["picture"];
+$what_to_order_input= $_POST["what_to_order"];
+$open_fire_input= $_POST["open_fire"];
```

Comment:
Definitely worth wrapping these in an isset to make sure some naughty user hasn't directly navigated to data.php

`if (isset($_POST['name']) {` will do the trick, no need to run all the POST items through isset